Figure integration tests are failing with "AssertionError: <qiskit_experiments.database_service.db_e[46 chars]7130> != b'hello world'" since Friday June 24

### Informations

- **Qiskit Experiments version**: latest (0349aa0241c7fae1b8d09f8098203bf2b27d7c7c)
- **Python version**: 3.8
- **Operating system**: Ubuntu Focal 20.04

### What is the current behavior?

I'm seeing figure-related tests failing since Friday June 24:

```
======================================================================

FAIL: test_add_figures (test.database_service.test_experiment_data_integration.TestExperimentDataIntegration)

test.database_service.test_experiment_data_integration.TestExperimentDataIntegration.test_add_figures

----------------------------------------------------------------------

testtools.testresult.real._StringException: Traceback (most recent call last):

  File "/home/travis/build/IBM-Q-Software/sw-ibmq-results/qiskit-experiments-main/test/database_service/test_experiment_data_integration.py", line 241, in test_add_figures

    self.assertEqual(rexp.figure(idx), hello_bytes)

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 912, in assertEqual

    assertion_func(first, second, msg=msg)

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 905, in _baseAssertEqual

    raise self.failureException(msg)

AssertionError: <qiskit_experiments.database_service.db_e[46 chars]7130> != b'hello world'

======================================================================

FAIL: test_add_figures_file (test.database_service.test_experiment_data_integration.TestExperimentDataIntegration)

test.database_service.test_experiment_data_integration.TestExperimentDataIntegration.test_add_figures_file

----------------------------------------------------------------------

testtools.testresult.real._StringException: Traceback (most recent call last):

  File "/home/travis/build/IBM-Q-Software/sw-ibmq-results/qiskit-experiments-main/test/database_service/test_experiment_data_integration.py", line 267, in test_add_figures_file

    self.assertEqual(rexp.figure(0), hello_bytes)

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 912, in assertEqual

    assertion_func(first, second, msg=msg)

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 905, in _baseAssertEqual

    raise self.failureException(msg)

AssertionError: <qiskit_experiments.database_service.db_e[46 chars]a5b0> != b'hello world'

======================================================================

FAIL: test_save_all (test.database_service.test_experiment_data_integration.TestExperimentDataIntegration)

test.database_service.test_experiment_data_integration.TestExperimentDataIntegration.test_save_all

----------------------------------------------------------------------

testtools.testresult.real._StringException: Traceback (most recent call last):

  File "/home/travis/build/IBM-Q-Software/sw-ibmq-results/qiskit-experiments-main/test/database_service/test_experiment_data_integration.py", line 323, in test_save_all

    self.assertEqual(hello_bytes, rexp.figure(0))

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 912, in assertEqual

    assertion_func(first, second, msg=msg)

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 905, in _baseAssertEqual

    raise self.failureException(msg)

AssertionError: b'hello world' != <qiskit_experiments.database_service.db_e[46 chars]a490>

======================================================================

FAIL: test_update_figure (test.database_service.test_experiment_data_integration.TestExperimentDataIntegration)

test.database_service.test_experiment_data_integration.TestExperimentDataIntegration.test_update_figure

----------------------------------------------------------------------

testtools.testresult.real._StringException: Traceback (most recent call last):

  File "/home/travis/build/IBM-Q-Software/sw-ibmq-results/qiskit-experiments-main/test/database_service/test_experiment_data_integration.py", line 276, in test_update_figure

    self.assertEqual(exp_data.figure(0), hello_bytes)

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 912, in assertEqual

    assertion_func(first, second, msg=msg)

  File "/opt/python/3.8.1/lib/python3.8/unittest/case.py", line 905, in _baseAssertEqual

    raise self.failureException(msg)

AssertionError: <qiskit_experiments.database_service.db_e[46 chars]d0d0> != b'hello world'
```

### Steps to reproduce the problem

Run the integration tests. Note that this repo has a cron job which should be running those but it's not (that's maybe a separate bug or should be fixed with this):

https://github.com/Qiskit/qiskit-experiments/runs/7066338031?check_suite_focus=true

### What is the expected behavior?

Integration tests should pass and kept up to date

### Suggested solutions

n/a

--- a/test/database_service/test_experiment_data_integration.py
+++ b/test/database_service/test_experiment_data_integration.py
@@ -264,7 +264,7 @@ class TestExperimentDataIntegration(QiskitTestCase):
 
         exp_data.add_figures(figures=file_name, save_figure=True)
         rexp = DbExperimentData.load(exp_data.experiment_id, self.experiment)
-        self.assertEqual(rexp.figure(0), hello_bytes)
+        self.assertEqual(rexp.figure(0).figure, hello_bytes)
 
     def test_update_figure(self):
         """Test updating a figure."""
@@ -320,7 +320,7 @@ class TestExperimentDataIntegration(QiskitTestCase):
         # so compare tags with a predictable sort order.
         self.assertEqual(["bar", "foo"], sorted(rexp.tags))
         self.assertEqual(aresult.result_id, rexp.analysis_results(0).result_id)
-        self.assertEqual(hello_bytes, rexp.figure(0))
+        self.assertEqual(hello_bytes, rexp.figure(0).figure)
 
         exp_data.delete_analysis_result(0)
         exp_data.delete_figure(0)

--- a/test/database_service/test_experiment_data_integration.py
+++ b/test/database_service/test_experiment_data_integration.py
@@ -273,7 +273,7 @@ class TestExperimentDataIntegration(QiskitTestCase):
         figure_name = "hello.svg"
 
         exp_data.add_figures(figures=hello_bytes, figure_names=figure_name, save_figure=True)
-        self.assertEqual(exp_data.figure(0), hello_bytes)
+        self.assertEqual(exp_data.figure(0).figure, hello_bytes)
 
         friend_bytes = str.encode("hello friend")
         exp_data.add_figures(

